docs/docs/self-host/env.mdx

```diff
@@ -57,6 +63,8 @@ growthbook:
     - API_HOST=http://<my ip or url>:4100
 ```
 
+Ensure you are rebuilding the container after changing any environment variables, e.g. `docker-compose up --build`. See the [docker compose build docs](https://docs.docker.com/engine/reference/commandline/compose_build/) for more information.
```

Comment:
I don't think this is required. The environment variables are specified at runtime, not build time. They just need to restart the container for it to take effect.